File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        dicc={}
        lista=[]
        frequent_num=[]
        for num in nums:
            if num not in dicc:
                dicc[num]=1
            else:
                dicc[num]+=1
        for num in dicc:
            lista.append((dicc[num],num))
        for x in range(k):
            m=max(lista)
            frequent_num.append(m[1])
            lista.remove(max(lista))
        return frequent_num

File: Data Structures & Algorithms/replace-elements-with-greatest-element-on-right-side/submission-1.java
class Solution {
    public int[] replaceElements(int[] arr) {
        int[] nums=new int[arr.length];
        int max=-1;

        for(int x=arr.length-1;x>=0;x--)
        {
            nums[x]=max;
            max=Math.max(max,arr[x]);
        }
        return nums;
    }
}